app/portainer/views/users/edit/userController.js

```diff
@@ -63,11 +63,24 @@ angular.module('portainer.app').controller('UserController', [
         });
     };
 
-    $scope.updatePassword = function () {
+    $scope.updatePassword = async function () {
+      let promise = Promise.resolve(true);
+      if (Authentication.getUserDetails().ID + '' === $transition$.params().id) {
+        promise = ModalService.confirmChangePassword();
+      }
```

Comment:
```
const isCurrentUser = Authentication.getUserDetails().ID === $scope.user.Id;
const confirmed = !isCurrentUser || await ModalService.confirmChangePassword();
```

than you can use `isCurrentUser` in line 79 too

Reply:
updated code~

Reply:
push?